packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -166,6 +185,9 @@ const ApplicationMenu = (props: ApplicationMenuProps) => {
           isActive={props.isActive}
           isExpanded={props.isMenuOpen}
           hasSubmenu={hasSubmenu}
+          topPosition={topPosition}
+          scrollTop={props.scrollTop}
```

Comment:
**nit**: I personally feel this a little weird. I would expect just a `topPosition` prop where we indicate to the `MenuItem` the vertical position it should use: we can then pass the calculation (`topPosition + props.scrollTop`) as the value.
I don't feel the `MenuItem` component needs to know both parameters as it only cares about what vertical position it should use.

What do you think?

Reply:
@CarlosCortizasCT yes, now that you've mentioned, I don’t know why I didn’t pass it this way in the first place. 😄 
[5b8e33d](https://github.com/commercetools/merchant-center-application-kit/pull/3169/commits/5b8e33dffa048ac4de7bfa9c1f0dac334d25223f)